metadata-service/iceberg-catalog/src/integrationTest/integration_test.py

```diff
@@ -181,7 +179,7 @@ def _test_basic_view_ops(spark_session):
     try:
         spark_session.sql("SELECT * FROM test_view")
         assert False, "test_view must not exist"
-    except:
```

Comment:
these should actually be using `pytest.raises(Exception, "expected message")` 

the way these tests are set up currently does not actually test anything, since the AssertionError would get caught and suppressed by the except block

Reply:
in other words - `ruff` is flagging a real issue